webapp/channels/src/components/profile_popover/profile_localtime.tsx

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {DateTime, Duration} from 'luxon';
+
+import {FormattedMessage} from 'react-intl';
+
+import {UserTimezone} from '@mattermost/types/users';
+import Timestamp from 'components/timestamp';
+import {getUserCurrentTimezone} from 'mattermost-redux/utils/timezone_utils';
+
+type ProfileTimezoneProps = {
+    profileUserTimezone?: UserTimezone;
+    currentUserTimezone: string | undefined;
+}
+
+const returnTimeDiff = (
+    currentUserTimezone: string | undefined | null,
+    profileUserTimezone: string,
+) => {
+    if (!currentUserTimezone) {
+        return null;
+    }
+    const currentUserDate = DateTime.local().setZone(currentUserTimezone);
+    const profileUserDate = DateTime.local().setZone(profileUserTimezone);
+
+    const offset = Duration.fromObject({
+        hours: (profileUserDate.offset - currentUserDate.offset) / 60,
+    });
+
+    if (!offset.valueOf()) {
+        return undefined;
+    }
+
+    const timeOffset = offset.toHuman({unitDisplay: 'short', signDisplay: 'never'});
+
+    return offset.valueOf() > 0 ? (
+        <FormattedMessage
+            id='user_profile.account.hoursAhead'
+            defaultMessage='({timeOffset} ahead)'
+            values={{timeOffset}}
+        />
+    ) : (
+        <FormattedMessage
+            id='user_profile.account.hoursBehind'
+            defaultMessage='({timeOffset} behind)'
+            values={{timeOffset}}
+        />
+    );
+};
+
+export const ProfileTimezone = (
+    {
+        currentUserTimezone,
+        profileUserTimezone,
+    }: ProfileTimezoneProps,
+) => {
+    const profileTimezone = getUserCurrentTimezone(profileUserTimezone) || 'UTC';
```

Comment:
@hmhealey - I notice you made a change to this in https://mattermost.atlassian.net/browse/MM-51359. I'm using it to get a timezone of the non-current user. Should I also adjust this?